index.js

```diff
@@ -33,13 +33,13 @@
 //. <pre>
 //:  Setoid   Semigroup   Foldable        Functor      Contravariant
 //: (equals)   (concat)   (reduce)         (map)        (contramap)
-//:               |           \         / | | | | \
-//:               |            \       /  | | | |  \
-//:               |             \     /   | | | |   \
-//:               |              \   /    | | | |    \
-//:               |               \ /     | | | |     \
-//:            Monoid         Traversable | | | |      \
-//:            (empty)        (traverse)  / | | \       \
+//:    |          |           \         / | | | | \
+//:    |          |            \       /  | | | |  \
+//:    |          |             \     /   | | | |   \
+//:    |          |              \   /    | | | |    \
+//:    |          |               \ /     | | | |     \
+//:   Ord      Monoid         Traversable | | | |      \
+//:  (lte)     (empty)        (traverse)  / | | \       \
```

Comment:
Nit: For "even" words it appears that we're pointing to the character half a character to the right of centre (the second `o` in `Monoid`, for example). Let's make this line point to the `o` in `Setoid`. :)